eng/ci/official-release.yml

```diff
@@ -0,0 +1,112 @@
+parameters:
+- name: publishToNugetOrg
+  displayName: Publish to nuget.org?
+  type: boolean
+  default: false
+- name: packages
+  displayName: Packages to publish
+  type: string
+  default: core
+  values:
+  - core
+  - sdk
+
+pr: none
+trigger: none
+
+resources:
+  repositories:
+  - repository: 1es
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+  - repository: eng
+    type: git
+    name: engineering
+    ref: refs/tags/release
+  pipelines:
+  - pipeline: build
+    source: dotnet-worker.official
+
+variables:
+  artifact_name: NugetPackages
+  ${{ if eq(parameters.packages, 'core') }}:
+    target_folder: azure-functions/dotnet/worker
+    package_name: Microsoft.Azure.Functions.Worker
+    package_pattern: |
+      Microsoft.Azure.Functions.Worker.*.nupkg
+      !Microsoft.Azure.Functions.Worker.Sdk.*.nupkg
+      !*.symbols.nupkg
+  ${{ if eq(parameters.packages, 'sdk') }}:
+    target_folder: azure-functions/dotnet/worker-sdk
+    package_name: Microsoft.Azure.Functions.Worker.Sdk
+    package_pattern: |
+      Microsoft.Azure.Functions.Worker.Sdk.*.nupkg
+      !*.symbols.nupkg
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc
+      image: 1es-ubuntu-22.04
+      os: linux
+
+    stages:
+    - stage: Validate
+      jobs:
+        - job: Validate
+
+          variables:
+            drop_path: $(Pipeline.Workspace)/drop
+
+          templateContext:
+            type: releaseJob
+            isProduction: true
+            inputs:
+            - input: pipelineArtifact
```

Comment:
Did not quite understand what is this `validate` doing and the inputs mentioned here?

Reply:
Added a comment and changed the name.